maps/aurora-2/aurora-8_derelict.dmm

```diff
@@ -7998,6 +7995,15 @@
 /obj/machinery/the_singularitygen,
 /turf/simulated/floor/airless,
 /area/space)
+"xa" = (
+/turf/simulated/floor/airless,
+/area)
+"xb" = (
+/turf/space,
```

Comment:
This one and the one below it don't seem right. Why not reuse the same key value?